contest_manager.py

```diff
@@ -41,8 +23,81 @@ def __init__(self, client, dmoj_client : DmojClient):
         self._users_db = MicroDB("users")
         self._contests_db = MicroDB("contests")
 
+    '''
+    @commands.command()
+    async def force_add(self, ctx):
+        self._users_db.set("Sardor", "donkogronko")
+
     @commands.command()
-    async def create_contest(self, ctx, contest_name, active, registrationRequired, endDate, force, *problem_links):
+    async def print_db(self, ctx):
+        await ctx.send(json.dumps(self._contests_db.get_all(), indent=4))
+    '''
+    
+    @commands.command()
+    async def create_weekly(self, ctx, week_number, *problem_links):
+        contest_name = f"Weekly Problem Set #{week_number}"
+        end_date = datetime.datetime.now() + datetime.timedelta(days=7)
```

Comment:
This should probably be a standardized due date (friday nights) not 7 days from creation time.